### PR TITLE
Rd 5023 2

### PR DESCRIPTION
**Pull request by ashish1605**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
             steps {
-                script {
-                    sh "ls"
-                    
-                    sh "packer build -var-file=variables.json packer-rhel7-ami.json"
-                    
+                withCredentials([file(credentialsId: 'hailstone_secrets', variable: 'secrets')]) {
+
+                   // sh 'docker run -e REGION=$region --env-file=$secrets -i -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd)/setup.sh:/setup.sh -v $(pwd)/packer-rhel7-ami.json:/template.json -v $(pwd)/variables.json:/variables.json hashicorp/packer:light build -var-file=variables.json template.json'
+                    sh 'docker run -e REGION=$region --env-file=$secrets -i -v /var/run/docker.sock:/var/run/docker.sock -v "$(pwd)"/:/workspace/ -w /workspace/ hashicorp/packer:light build -var-file=variables.json packer-rhel7-ami.json'
```
Comment (ashish1605) [inline Jenkinsfile]: @grugnog I have added quotes as required. The PR seems to be failing again with tests like previously it did.

File: packer-rhel-ami.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    },
+    "builders": [{
+      "type": "amazon-ebs",
+      "source_ami_filter": {
+        "filters": {
+          "virtualization-type": "hvm",
+          "name": "RHEL-7.7_HVM_GA-20190723-x86_64-1-Hourly2-GP2",
+          "root-device-type": "ebs"
+        },
+        "owners": ["309956199498"],
```
Comment (aabond) [inline packer-rhel-ami.json]: @ashish1605 Do we want this value hard-coded or made available via variable?

File: packer-rhel7-ami.json [hunk 3]
```diff
@@ -1,46 +1,44 @@
...
-      "instance_type": "{{user `instance_type`}}",
-      "ssh_username": "{{user `ssh_username`}}",
-      "ssh_pty": "true",
-      "ami_name": "RHEL7-hardened-{{timestamp}}",
-      "ssh_keypair_name": "{{user `keypair_name`}}",
-      "iam_instance_profile": "{{user `iam_instance_profile`}}",
-      "ssh_private_key_file": "{{user `key_file_path`}}",
-      "associate_public_ip_address": true
-      
+      "owners": ["309956199498"],
```
Comment (aabond) [inline packer-rhel7-ami.json]: @ashish1605 Do we want this value hard-coded or made available via variable?

File: packer-rhel-ami.json (hunk 2 above)
Comment (ashish1605) [reply packer-rhel-ami.json]: @aabond This is publicly available on AWS. We can leave this value as hard-coded

File: packer-rhel7-ami.json (hunk 3 above)
Comment (ashish1605) [reply packer-rhel7-ami.json]: @aabond  This is publicly available on AWS. We can leave this value as hard-coded

File: setup.sh [hunk 4]
```diff
@@ -27,6 +24,17 @@ cd ~
 OVAL_REPORT_NAME=${OS}-oval-report.html
 REPORT_NAME=${OS}-${SCAP_TARGET}-report.html
 
+# Remediation steps
+firewall-cmd || yum install firewalld -y
+systemctl start firewalld
+firewall-cmd --set-default-zone public
```
Comment (aabond) [inline setup.sh]: firewall-cmd --set-default-zone drop
firewall-cmd --zone=drop --permanent --add-service=ssh

**Pull request merged**